### ISSUE TITLE
no-dynamic-styling rule false positive on _pseudo conditions

### ISSUE THREAD

**Issue by jagregory** [chakra-ui/eslint-plugin-panda#11]
I'm seeing false positives with the `no-dynamic-styling` rule on a codebase which uses _ conditions. For example:

```ts
export const tab = cva({
  base: {
    _selected: {
      color: 'tabList.color',
      fontWeight: 'heavy',
    },

    _focus: {
      outline: 'active',
    },
  },
```

The `_selected` and `_focus` lines both are flagged as errors. I've seen this with styles using all the style functions, `css`, `sva`, and `sva`.

<img width="719" alt="Screen Shot 2024-02-01 at 9 10 17 am" src="https://github.com/chakra-ui/eslint-plugin-panda/assets/10828/bc0f1962-72d6-4a00-b7fd-55952634d789">
